### ISSUE TITLE
Use wallet_addEthereumChain with MetaMask for other chains

### ISSUE THREAD

**Issue by ilanolkies** [rsksmart/rLogin#276]
TLDR; Proposal: allow developer to configure network settings to pass to `wallet_addEthereumChain` when user is stepping on _wrong network_ UX

---

### About what to do with MetaMask `wallet_addEthereumChain`

See this related comment https://github.com/rsksmart/rLogin/issues/268#issuecomment-1135384736

~I know this is not directly what we are discussing here but ~I bring up similar concerns. I would like to remark some:
- The idea of rLogin is to make easy to build dapps on RSK, that is why we started by implementing just that two networks. Some dapps are cross/multi-chain (like token bridge) so we started exploring on different networks while keeping "a generic UX" for them. Now I see a big change: we are moving away from generic UX into something more custom, we need to be careful 
- We want to keep the tool decentralized thus allow the developer to choose whatever node they want their users to use. A clear example of this is. See for example https://moralis.io/how-to-connect-to-polygon-nodes/
- We know insights about how RSK infra works, if we want to add refs to infra of other networks we will need to do further research. See for example https://docs.polygon.technology/docs/develop/network-details/network/
  
  > Public RPCs may have traffic or rate-limits depending on usage. You can sign up for a dedicated free RPC URL at the following
  
  This can become a bummer "I connected to Polygon with rLogin and I don't see my balance updated"

_Originally posted by @ilanolkies in https://github.com/rsksmart/rLogin/pull/273#discussion_r880516686_
